src/tmx/TmxUtils/src/kafka/kafka_consumer_worker.cpp

```diff
@@ -134,7 +141,7 @@ namespace tmx::utils
         switch (message->err())
         {
         case RdKafka::ERR__TIMED_OUT:
-            FILE_LOG(logWARNING) << _consumer->name() << " consume failed: " <<  message->errstr() << std::endl;
+            FILE_LOG(logDEBUG4) << _consumer->name() << " consume failed: " <<  message->errstr() << std::endl;
```

Comment:
What do you think about setting this back to log WARNING and just setting the timeout value large to something that indicates an error like 60s or something.

Reply:
I think it will still print the log constantly when we are in between testing and no data is consumed during the time.

Reply:
I do not think we have any kafka messages that have intervals even close to 60s so it should never print unless we are not receiving kafka messages

Reply:
yeah, It could happen as we stop sending BSM/MP/MOM when the vehicle disengage and before the next round of testing. The while loop in the message consumption will keep checking if there are data from those topics.

Reply:
Isn't this unique to consumers.

Reply:
I think so, consumers needs to continuously poll Kafka for data with the while loop. Every poll request, the consumer is telling the Kafka broker that it is alive.